src/coprocessor/dag/batch/executors/simple_aggr_executor.rs

```diff
@@ -61,6 +61,10 @@ impl BatchSimpleAggregationExecutor<Box<dyn BatchExecutor>> {
     pub fn check_supported(descriptor: &Aggregation) -> Result<()> {
         assert_eq!(descriptor.get_group_by().len(), 0);
         let aggr_definitions = descriptor.get_agg_func();
+        if aggr_definitions.is_empty() {
```

Comment:
Although this should not happen, can we still accept it?

Reply:
I don't know what will happen to simple aggregation executor if there is no column :) Maybe it is ok, maybe it will break. So I simply rejected this case.

Reply:
Ok, we can check it later.